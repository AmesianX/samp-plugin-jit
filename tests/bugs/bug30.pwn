#include <a_samp>

forward Test();

public OnGameModeInit() {
	CallLocalFunction("Test", "");
	printf("CallLocalFunction returned");
}

main() {}

public Test()
{
	new a[2];
	new b = -1;
	a[b] = 0;

	print("Compile this script with array bounds checks turned on");
}
