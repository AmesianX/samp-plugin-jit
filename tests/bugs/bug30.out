  JIT plugin v.* is OK\.
  Loaded\.
 Loaded 1 plugins.


Started server on port: 7777, with maxplayers: 500 lanmode is OFF\.


Filterscripts
---------------
  Loaded 0 filterscripts\.

CallLocalFunction returned
Number of vehicle models: 0
